#158487957 Integrate HoundCI with badge attached to readme

nouwatinjacob opened the pull request:
#### What does this PR do?
- It integrates houndci and add badge to reame
#### Description of Task to be completed?
- It works with eslint to check if the pull request code conform to eslint rules
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#158487957
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

nouwatinjacob committed:
chore(configure-eslint):

- install dependencies for eslint and airbnb
- setup the rules for eslint

[finishes  #158487952]

nouwatinjacob committed:
chore(Integrate HoundCI):

- Integrate HoundCI with badge attached to readme

[finishes #158487957]

andela-moe commented on .hound.yml:
```diff
@@ -0,0 +1,3 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
```
You should have a new line at the end of this file

nouwatinjacob replied:
New line added

andela-moe commented on README.md:
```diff
@@ -1,3 +1,5 @@
+[![Continuous Integration](https://camo.githubusercontent.com/23ee7a697b291798079e258bbc25434c4fac4f8b/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f50726f7465637465645f62792d486f756e642d6138373364312e737667)](https://houndci.com)
+
```
Even though HoundCI does not provide a badge, I admire your initiative of using a static badge. Good work.

nouwatinjacob replied:
Thanks

andela-moe commented on .eslintrc:
```diff
@@ -0,0 +1,34 @@
+        }
+    }]
+  }
+}
```
This seems to be part of another PR. Any reason why it is here also?

andela-moe reviewed (commented):
Please review comments and resolve where necessary.

nouwatinjacob committed:
chore(Configure Eslint):

- removed eslintrc file
- add new line at the end of hound.yml file

[Finishes #158487957]

nouwatinjacob committed:
new line added to hound.yml

nouwatinjacob committed:
chore(Configure-Eslint):

- add new line

[Fixes #158487957]
